src/parsed_codeblock.py

```diff
@@ -0,0 +1,31 @@
+from enum import Enum
+from pathlib import Path
+
+class CodeblockSeparator(Enum):
+    BACKTICKS = 1
+
+class ParsedCodeBlock:
+    def __init__(self):
+        self.completed: bool = False
+        self.content: str = None
+        self.start_pos: int = None
+        self.end_pos: int = None
+        self.separator: CodeblockSeparator = CodeblockSeparator.BACKTICKS
```

Comment:
didn't use that, delete

create issue for multiple CodeblockSeparator support -> https://github.com/Abb4/codeblock-to-anki/issues/6